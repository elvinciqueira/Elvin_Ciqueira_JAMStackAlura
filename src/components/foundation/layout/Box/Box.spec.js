/* eslint-disable no-undef */
import React from 'react'

import {Box} from './'

describe('<Box />', () => {
  test('it renders', () => {
    const tree = rendererCreateWithTheme(<Box />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('width and height props set width/height', () => {
    const tree = rendererCreateWithTheme(
      <Box width="50%" height="50%" />,
    ).toJSON()

    expect(tree).toMatchSnapshot()
    expect(tree).toHaveStyleRule('width', '50%')
    expect(tree).toHaveStyleRule('height', '50%')
  })
})
