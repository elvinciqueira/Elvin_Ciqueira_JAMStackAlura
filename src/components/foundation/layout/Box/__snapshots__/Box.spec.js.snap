// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> it renders 1`] = `
<div
  className=""
/>
`;

exports[`<Box /> width and height props set width/height 1`] = `
.c0 {
  width: 50%;
  height: 50%;
}

<div
  className="c0"
  height="50%"
  width="50%"
/>
`;
